service/service.go

```diff
@@ -146,6 +149,37 @@ func New(config Config) (*Service, error) {
 			return nil, microerror.Mask(err)
 		}
 	}
+	var newWatcherFactory informer.WatcherFactory
+	{
+
+		zeroObjectFactory := &informer.ZeroObjectFactoryFuncs{
+			NewObjectFunc: func() runtime.Object {
+				var pod api.Pod
+				return &pod
+			},
+			NewObjectListFunc: func() runtime.Object {
+				var podList api.PodList
```

Comment:
This did not work without declaring a variable first, as a struct can't be returned. Is there a better way to do this?